pyproject.toml

```diff
@@ -43,12 +44,6 @@ ruff = [
 [project.scripts]
 python-typing-update = "python_typing_update.__main__:main"
 
-[tool.setuptools]
-license-files = ["LICENSE", "AUTHORS"]
-
-[tool.setuptools.package-data]
-"*" = ["py.typed", "*.pyi"]
```

Comment:
`py.typed` and `*.pyi` files are included by default.
https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html#controlling-files-in-the-distribution